test/integration/builder/transaction.js

```diff
@@ -659,5 +659,34 @@ module.exports = function(knex) {
         return builder;
       });
     });
+
+    it('#3690 does not swallow exception when error during transaction occurs', async () => {
```

Comment:
We actually should be using standard functions instead of arrow functions when creating `mocha` test cases.  Otherwise, we'll actually lose some `mocha` functionality:

https://remarkablemark.org/blog/2016/08/20/arrow-functions-in-mocha/

Reply:
For this.skip()?

Reply:
Exactly.